ios/SecuritySdk/Classes/Digital Signature/DigitalSignatureVerifier.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DigitalSignatureVerifier.swift
+//  SecuritySdk
+//
+
+import Foundation
+
+/// Verifier to perform digital signature verification operations
+public protocol DigitalSignatureVerifier {
+    /**
+     Verifies a blob of data with a supplied signature using the underlying verifying key
```

Comment:
at present we'll need to have one of these that supports verifying tink keys, based on a provided trifle certificate. Could you file a jira task for this as a follow-on?  Also, do you have any concept of how difficult that will be?

Reply:
it will just be a parallel class that conforms to this protocol. how difficult it would be depends on the tink library itself. I've not worked with the obj-c? / swift version of it before